extensions/wikia/SpecialNewWikis/SpecialNewWikis_body.php

```diff
@@ -195,17 +195,17 @@ function reallyDoQuery( $offset, $limit, $descending ) {
 	function formatRow( $row ) {
 		global $wgLang;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "$wgLang" local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1481)